apps/ewallet_config/test/ewallet_config/utils/types/wallet_address_test.exs

```diff
@@ -43,25 +43,26 @@ defmodule EWalletDB.Types.WalletAddressTest do
     end
   end
 
-  describe "wallet_address/2" do
-    test "populates the schema with a valid wallet address" do
-      {:ok, wallet} =
-        :wallet
-        |> params_for(address: nil)
-        |> Wallet.insert()
-
-      assert String.match?(wallet.address, ~r/^[a-z]{4}[0-9]{12}$/)
-    end
-
-    test "uses the given wallet address if provided" do
-      {:ok, wallet} =
-        :wallet
-        |> params_for(address: "test-1234-5678-9012")
-        |> Wallet.insert()
-
-      assert wallet.address == "test123456789012"
-    end
-  end
+  #
```

Comment:
Should we re-enable these tests?

Reply:
No they have to be moved to another test file actually, or be deleted completely. Since they interact with the database and assume this module is being used a part of a schema, it shouldn't be here. This is testing the integration of the `WalletAddress`, not the `WalletAddress` itself.